core/controllers/editor.py

```diff
@@ -889,43 +888,35 @@ def post(self, exploration_id):
         if not raw:
             raise self.InvalidInputException('No image supplied')
 
-        file_format = imghdr.what(None, h=raw)
```

Comment:
I have one concern about removing this: in general, in web dev, there is a principle to not trust anything the frontend sends to the backend, since spoofing can happen. So if we want to validate anything we will need to do so in the backend. It is a pity that this validation has false negatives, though. Is it possible to validate another way, or at the very least do some validation to check that the file is indeed an image (even if it's a different type of image)?